janitor/io.py

```diff
@@ -69,3 +71,45 @@ def read_csvs(
         )
     else:
         return dfs_dict
+
+
+def read_commandline(cmd: str) -> pd.DataFrame:
+    """
+    Read a CSV file based on a command-line command.
+
+    For example, you may wish to run the following command on `sep-quarter.csv`
+    before reading it into a pandas DataFrame:
+
+    ```bash
+    cat sep-quarter.csv | grep .SEA1AA
+    ```
+
+    In this case, you can use the following Python code to load the dataframe:
+
+    ```python
+    import janitor as jn
+    df = jn.read_commandline("cat sep-quarter.csv | grep .SEA1AA")
+    ```
+
```

Comment:
In reviewing the docstring, I think it is important to communicate to end-users the following:

"This function assumes that your command line command will return an output that is parsable using pandas.read_csv and StringIO. We default to using pd.read_csv underneath the hood. Keyword arguments are passed through to read_csv."